srcpkgs/libopenmpt/template

```diff
@@ -1,33 +1,32 @@
 # Template file for 'libopenmpt'
 pkgname=libopenmpt
-version=0.5.4
+version=0.5.5
 revision=1
 wrksrc="libopenmpt-${version}+release.autotools"
 build_style=gnu-configure
-configure_args="$(vopt_with pulseaudio)
- $(vopt_with sdl) $(vopt_with sdl2)
- $(vopt_enable libopenmpt_modplug) $(vopt_enable libmodplug)"
-hostmakedepends="pkg-config"
-makedepends="zlib-devel mpg123-devel libogg-devel libvorbis-devel
- portaudio-devel portaudio-cpp-devel libsndfile-devel libflac-devel
+configure_args="$(vopt_with pulseaudio) $(vopt_with sdl2) $(vopt_with portaudio)"
+hostmakedepends="pkgconf"
+makedepends="zlib-devel libsndfile-devel
+ libflac-devel mpg123-devel libogg-devel libvorbis-devel
  $(vopt_if pulseaudio pulseaudio-devel) $(vopt_if sdl2 SDL2-devel)
- $(vopt_if sdl SDL-devel)"
+ $(vopt_if portaudio portaudio-devel) $(vopt_if portaudio portaudio-cpp-devel)"
 short_desc="Cross-platform C & C++ library to decode tracked music files (modules)"
 maintainer="a dinosaur <nick@a-dinosaur.com>"
 license="BSD-3-Clause"
 homepage="https://lib.openmpt.org/libopenmpt/"
 distfiles="https://lib.openmpt.org/files/libopenmpt/src/libopenmpt-${version}+release.autotools.tar.gz"
-checksum=f34d06b9daa7bca111625369e5bbc2e7c0e0e04737a439b0e6320811babcef40
-conflicts="$(vopt_if libmodplug libmodplug)"
+checksum=f1e01483ebf1a680d9ec030c9af20f5f2a5ac0f1e0642c18bd5593cfaa9ceb6b
 
 post_install() {
 	vlicense LICENSE
 }
 
 # Package build options
-build_options="pulseaudio sdl2 sdl libopenmpt_modplug libmodplug"
-build_options_default="pulseaudio"
-vopt_conflict sdl2 sdl
+build_options="pulseaudio sdl2 portaudio"
+build_options_default="pulseaudio sdl2 portaudio"
+desc_option_pulseaudio="Enable PulseAudio driver in openmpt123"
+desc_option_sdl2="Enable SDL2 driver in openmpt123"
```

Comment:
These two are described in `common/options.description` already, you can remove them.

Reply:
Removed those and changed the portaudio line to be more consistent.